cmd/cvp-trigger/main.go

```diff
@@ -223,6 +223,13 @@ func main() {
 	if o.targetNamespaces != "" {
 		envVars[steps.OOTargetNamespaces] = o.targetNamespaces
 	}
+	input := strings.Builder{}
+	input.WriteString("--input-hash=")
+	for key, value := range envVars {
+		input.WriteString(key)
+		input.WriteString(value)
+	}
```

Comment:
/paranoid

This will still fail if CVP submits two identical jobs (same bundle, same base OCP...) in the same time window.  @dirgim is that possible?

Reply:
Submitting two identical jobs in the same time window should not be possible in general, at the very least the index image references should be different.
The only exception I can think of would be if the cvp-trigger loses connection to the Prow cluster and CVP retries re-running the job right after it.

Reply:
/hold cancel

Reply:
/paranoid cancel

Reply:
/lgtm

Reply:
/hold

oh wait, @stevekuznetsov this needs sorting to not flake tests